transforms.py

```diff
@@ -205,7 +205,7 @@ def __init__(self):
         # However, this is eliminated above while filtering out images without
         # annotations. Thus, we fake it here
         mock_dataset = SimpleNamespace(ids=["invalid"])
-        wrapper = wrapper_factory(mock_dataset)
+        wrapper = wrapper_factory(mock_dataset, target_keys=None)
```

Comment:
This gives v2 an "unfair advantage" due to pytorch/vision#7489. If we want a fairer comparison, we can do

```suggestion
        wrapper = wrapper_factory(mock_dataset, target_keys={"boxes", "masks", "labels"})
```

However, this means we shouldn't report these numbers for v2 performance, since we are artificially throttling it here.